rts/lra/test/basic/pom.xml

```diff
@@ -60,4 +60,27 @@
 
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <jvm.args.modular>
```

Comment:
@mayankkunwar please can you add a comment indicating why these settings are needed?

Reply:
okay

Reply:
Is it just needed by the tests? Or is there something in the core that needs reflection? For example `--illegal-access=permit` - is that needed in production?

Reply:
`--illegal-access=permit` will not be needed, I used this just for testing. Later I will remove this

Reply:
Later I will also remove other arguments which are not required.